protocols/gossipsub/src/error.rs

```diff
@@ -1,153 +1,33 @@
-// Copyright 2020 Sigma Prime Pty Ltd.
+// TODO: which copyright here?
```

Comment:
Protocol labs, or other?

Reply:
Given that this all new content, please use the Protocol Labs copyright header.